pkg/sql/importer/import_stmt_test.go

```diff
@@ -3836,6 +3839,22 @@ func TestImportDefaultWithResume(t *testing.T) {
 	defer TestingSetParallelImporterReaderBatchSize(batchSize)()
 	defer row.TestingSetDatumRowConverterBatchSize(2 * batchSize)()
 
+	ctx, cancel := context.WithTimeout(context.Background(), testutils.RaceSucceedsSoonDuration)
```

Comment:
this feels odd to me: you can get this (dumping all stacks after 45s) by just running with `-timeout 45s` can't you

Reply:
Thanks for reviewing!

Good point, removed the timeout here.